server/build.gradle.kts

```diff
@@ -79,11 +79,11 @@ val copyDockerFolder: TaskProvider<Copy> =
         into(dockerBuildRootDirectory)
     }
 
-// @todo(#343) - createProductionDotEnv is temporary and used by the suites,
+// @todo(#343) - createDotEnv is temporary and used by the suites,
 //  once the suites no longer rely on the .env file from the build root we
 //  should remove this task
-val createProductionDotEnv: TaskProvider<Exec> =
-    tasks.register<Exec>("createProductionDotEnv") {
+val createDotEnv: TaskProvider<Exec> =
```

Comment:
Changed to createDotEnv for several reasons:
1. We have similar task in simulator and they essentially do the same thing (create env file). So i wanted to match the names.
2. This is temporary here.
3. Those are default, not production values, but moreover it does not make sense at all to name the simulator task `createProductionDotEnv`.

Reply:
This was added as a workaround because the suites rely on a built .env of the server to run (they need the version from there to build the correct test container). The only way to do that (as a workaround at that time) was to create this simple task in order to do `dependsOn` that task. I suggest we do not touch that and we simply call the local to the simulator `update-env.sh` directly form the `commandline` of gradle inside the `startDockerContainer` task. In this way we will not have these changes here and we will have one less task in the gradle file of the simulator and we are completely isolated.